utils/sccli.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# sccli - Service Change CLI
+# Script to switch openshift from k8s or vice-versa 
+
+function single_node_setup ()
```

Comment:
This function name should be changed to single_node_kube_setup() as this function only handles k8s